content/en/index.md

```diff
@@ -14,11 +14,12 @@ It’s free to use for all developers and software providers, with Sigstore’s
 
 ## How to use Sigstore
 
+* Install Cosign - [Cosign Installation page](/cosign/installation/)
 * Sign software artifacts with [Cosign](/cosign/overview/) — [Quick start](/cosign/overview/#quick-start)
 * Sign Git commits with [Gitsign](/gitsign/overview/) - [Quick start](/gitsign/overview/#quick-start)
-* Verify entries with [Cosign](/cosign/verify/)
 * Use the [policy controller](/policy-controller/overview/) to enforce Kubernetes policies
-* Use the [Fulcio Certificate Authority](/fulcio/overview/)
```

Comment:
Do we not want Fulcio mentioned at all here?

Reply:
My thought was no, for this section. Operating Fulcio/Rekor would fall under developer documentation, rather than user documentation. Maybe a separate section linking to overview pages for Rekor and Fulcio, for "developers or private deployment operators"?

Reply:
Agreed, this section works well for getting end users onto the correct user journey; I don't think we should mix user and developer docs.

Reply:
I can put in a link to the overviews of Fulcio and Rekor.   Shall we go with that?  or is that mixing user and developer at this point still?

Reply:
I think it may be worthwhile to say that Fulcio is the CA, but we can do that later.